tests/end-to-end/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
     id("net.serenity-bdd.serenity-gradle-plugin") version "4.0.1"
 }
 
-group = "io.iohk.atala.prism"
+group = "org.hyperledger.identus"
```

Comment:
It's better to keep these changes until the legal approves moving the repo under the Hyperledger org.
Otherwise, it might be difficult to release a new package: each group maintains the secrets that allow them to publish packages.

Reply:
this is the automation package, there's no artifact being released from here... but should I revert this change @yshyn-iohk ?